extension/gds/src/include/function/config/louvain_config.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <string>
+
+#include "common/exception/binder.h"
+#include "common/types/types.h"
+#include "function/gds/gds.h"
+
+namespace kuzu {
+namespace function {
+
+// The maximum number of phases in which the graph is clustered and then aggregated.
+struct MaxPhases {
+    static constexpr const char* NAME = "maxphases";
+    static constexpr common::LogicalTypeID TYPE = common::LogicalTypeID::INT64;
+    static constexpr int64_t DEFAULT_VALUE = 20;
+
+    static void validate(int64_t maxPhases) {
+        if (maxPhases < 0) {
```

Comment:
Can the type be `LogicalTypeID::UINT64` in which case we can skip the validation?

Reply:
Yes, but we haven't implemented the casting for optional parameters.
If you change to  uint64, users have to explicitly cast the maxphases

Reply:
Got it, thanks.